src/main/java/org/entando/plugins/pda/pam/summary/RequestsProperties.java

```diff
@@ -8,7 +8,8 @@
 @Configuration
 @PropertySource("classpath:requests.properties")
 @Getter
-public class RequestsProperties {
+@SuppressWarnings("PMD.TooManyFields")
+public class RequestsProperties  {
```

Comment:
I think it would be better to create a DualTimeSeriesProperties class and move the related properties to this new class (RequestsProperties was created for the Requests summary).